jellyfin-core/src/androidMain/kotlin/org/jellyfin/sdk/JellyfinOptions.kt

```diff
@@ -0,0 +1,29 @@
+package org.jellyfin.sdk
+
+import android.content.Context
+import org.jellyfin.sdk.android.androidDevice
+import org.jellyfin.sdk.model.ClientInfo
+import org.jellyfin.sdk.model.DeviceInfo
+
+public actual data class JellyfinOptions(
+	public val context: Context,
+	public actual val clientInfo: ClientInfo?,
+	public actual val deviceInfo: DeviceInfo?,
+) {
+	public actual class Builder {
+		public var context: Context? = null
+		public var clientInfo: ClientInfo? = null
+
+		public actual fun build(): JellyfinOptions {
+			requireNotNull(context) { "An Android context is required when using the jellyfin-android platform." }
+
+			return JellyfinOptions(
+				context!!,
+				clientInfo,
+				deviceInfo = androidDevice(context!!)
+			)
```

Comment:
Could even use an expression body afterwards.

```suggestion
			return JellyfinOptions(
				requireNotNull(context) { "An Android context is required when using the jellyfin-android platform." },
				clientInfo,
				deviceInfo = androidDevice(context!!)
			)
```